### PR TITLE
Merge previous features

### PR DESCRIPTION
**Pull request by sarahyjja**
This PR has been created to integrate the previous features to the new project structure:
- Header particularities
- Old homepage 
- Footer
- Update tests and CSS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# laa-template-typescript
+# equiniti-historical-data
 [![repo standards badge](https://img.shields.io/badge/dynamic/json?color=blue&style=flat&logo=github&label=MoJ%20Compliant&query=%24.result&url=https%3A%2F%2Foperations-engineering-reports.cloud-platform.service.justice.gov.uk%2Fapi%2Fv1%2Fcompliant_public_repositories%2Fhmpps-template-typescript)](https://operations-engineering-reports.cloud-platform.service.justice.gov.uk/public-github-repositories.html#hmpps-template-typescript "Link to report")
 [![CircleCI](https://circleci.com/gh/ministryofjustice/hmpps-template-typescript/tree/main.svg?style=svg)](https://circleci.com/gh/ministryofjustice/hmpps-template-typescript)
```
Comment (psoleckimoj) [inline README.md:3]: The links here will want changing at some point as they link to CircleCI.

File: helm_deploy/values.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    elasticache-redis:
+      REDIS_HOST: "primary_endpoint_address"
+      REDIS_AUTH_TOKEN: "auth_token"
+
+  allowlist:
+    groups:
+      - internal
+
+generic-prometheus-alerts:
+  targetApplication: equiniti-historical-data
```
Comment (psoleckimoj) [inline helm_deploy/values.yaml:61]: Missing closing newline

File: server/views/pages/index.njk [hunk 3]
```diff
@@ -1,11 +1,105 @@
...
+                      action: '#',
+                      method: 'post',
+                      text: 'Search'
+                  }) }}
+              </div>
+          </main>
+
+      </div>
+
+  {% endblock %}
```
Comment (psoleckimoj) [inline server/views/pages/index.njk:105]: Missing closing newline

File: server/views/partials/footer.njk [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+                href: "#",
+                text: "Terms and conditions"
+            },
+            {
+                href: "#",
+                text: "Privacy settings"
+            }
+        ]
+    }
+}) }}
```
Comment (psoleckimoj) [inline server/views/partials/footer.njk:28]: Missing closing newline

File: README.md (hunk 1 above)
Comment (sarahyjja) [reply README.md]: Done! Link changed and linked to CI

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**